ilastik/workflows/carving/opCarving.py

```diff
@@ -263,29 +262,12 @@ def setupOutputs(self):
 
         self.AllObjectNames.meta.dtype = object
 
-    def hasCurrentObject(self):
-        """
-        Returns current object name. None if it is not set.
-        """
-        # FIXME: This is misleading. Having a current object and that object having
-        # a name is not the same thing.
-        return self._currObjectName
-
     def getCurrentObjectName(self):
+        f"""
```

Comment:
fstrings cannot be used as docstrings. See: https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals (right at the end of that paragraph)

Reply:
😅